sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -591,3 +591,49 @@
   font-family: "BCSans-bold", "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   color: $brand-blue;
 }
+
+.restrictions-details {
+  font-style: $font-style-normal;
+  font-weight: $font-weight-normal;
+  font-size: $font-size-xsmall;
+  line-height: $line-height-header-xsm;
+}
+
+/* Grid for info with icon to the left */
+.info-grid-icon {
+  grid-area: icon;
+  font-size: 24px;
+  color: $banner-color-info;
+}
+
+.info-grid-header {
+  grid-area: header;
+  font-style: $font-style-normal;
+  font-weight: $font-weight-bold;
+  font-size: $font-header-small;
+  line-height: $line-height-header-sm;
+  color: $secondary-header-color;
+}
+
+.info-grid-text {
+  grid-area: info;
```

Comment:
what does `grid-area: info;` do.

Reply:
Yes, it is.  Let me check this with Lynn.

Reply:
It was removed as per our conversation. Thanks.